bot/bot.go

```diff
@@ -21,12 +22,19 @@ var conn *badger.DB
 var botPrefix string
 var messages map[string]map[string]string
 
+// RegEx used to split all command parameters, considering anything between quotes as a single parameter.
+// Ex: `> move ThisChannel "That Channel"` will be processed as [">", "move", "ThisChannel", "That Channel"]
+var commandRegEx, _ = regexp.Compile(`(".*?"|\S+)`)
+
+// RegEx used to remove starting and ending quotes from the parameters
+var parameterQuotesRegEx, _ = regexp.Compile(`(^"|"$)`)
+
 // Close function ends the bot connection and closes its database
 func Close() {
 	log.Println("Closing")
-	commander.Close()
+	_ = commander.Close()
```

Comment:
Explicitly ignoring returned values makes things more verbose, but its not necessarily a bad thing. We might just remove the return values from some functions if we endup not using them.

Reply:
When possible, ignored errors should be avoided by explicitly defining them as ignored instead of leaving them unhandled as the latter is probably a bug!